celery_config.py

```diff
@@ -107,7 +107,9 @@ def init_celery_tracing(*args, **kwargs):
 
 # Backfill GH Apps
 backfill_existing_gh_app_installations_name = "app.tasks.backfill_existing_gh_app_installations.BackfillExistingGHAppInstallationsTask"
+backfill_existing_individual_gh_app_installation_name = "app.tasks.backfill_existing_individual_gh_app_installation.BackfillExistingIndividualGHAppInstallationTask"
```

Comment:
this is more for future reference, but very nit

The task name is (or should be) formatted as
`app.[tasks|cron].<TaskConfigGroup>.<task name>`

so all of the tasks in the backfill could have the same <TaskConfigGroup> of `backfill` or `backfil_github_apps`, for example. This makes it easier for us to know what tasks are related.

Reply:
Ty for this! That honestly makes a lot more sense 🙏